lua/starfall/libs_sh/entities.lua

```diff
@@ -2029,6 +2030,24 @@ return function(instance)
 		Ent_SetFlexScale(ent, scale)
 	end
 
+	--- Gets the acceptable value range for the flex controller, as defined by the model
+	-- @shared
+	-- @param number flexid The id of the flex
+	-- @return The minimum value for this flex
+	-- @return The maximum value for this flex
+	function ents_methods:getFlexBounds(flexid)
+		local ent = getent(self)
+	    	
+		checkluatype(flexid, TYPE_NUMBER)
+	    	flexid = math.floor(flexid)
```

Comment:
nit: formatting messed up here, I probably should add a vscode recommended extensions so that stylua always runs